snapshot.yaml

```diff
@@ -41,3 +39,9 @@ packages:
   # https://github.com/EdutainmentLIVE/recurly
   - git: https://github.com/EdutainmentLIVE/recurly
     commit: 9c8badd97c662338ed6f1c201df329a3d9e5cd5a
+
+  # mongoDB 2.13.0.2 was not uploaded to hackage for whatever reason
```

Comment:
Might be worth bugging them about this.

Reply:
We found out why it wasn't uploaded 😅 it just doesn't work.

Reply:
Is this the reason?

Reply:
https://github.com/yesodweb/persistent/issues/1284

Reply:
Is our current game plan still to stick with Persistent 2.12?

Reply:
Going to see if I can get persistent compiling, otherwise I'm just going to leave the snapshot alone.